core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -357,6 +371,7 @@ def usage():
                            '--command [%s] --study_directory <path to directory> '
                            '--meta_filename <path to metafile>'
                            '--data_filename <path to datafile>'
+                           '--study_ids <cancer study ids, comma separated>'
```

Comment:
maybe we should mention to the user that this is used in conjunction with the remove-study command? Otherwise some may think this must be supplied during an import.

Reply:
second that. Really `command` should be a proper subcommand instead of an option. Each command should have it's own options

Reply:
I created an issue for this: https://github.com/cBioPortal/cbioportal/issues/7201. It would be good tho to mention like Rob said that this is only for the remove-study command

Reply:
Refactoring this script so that options are context-sensitive to the command could be an issue with the "good first issue" tag. But we also are hoping to replace the importer and scripts package entirely with a standalone code base (this year?)

Reply:
Thanks for the context @sheridancbio ! Since it would be a pretty minor change I think it can be ok for a first issue even tho we might decide to retire the code altogether, but that's def good to know!

Reply:
My suggestion would be to change this line to:
'--study_ids <cancer study ids for remove-study, comma separated>'

Reply:
@inodb @sheridancbio I have updated the user help lines in the script as per your suggestion.